fxa/tests/__init__.py

```diff
@@ -1,3 +1,7 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from requests.packages import urllib3
+urllib3.disable_warnings()
```

Comment:
I worry about accidentally disabling these in production by importing `fxa.tests` without realizing it.  Can we put this into a module-level setup/teardown function so it only applies for the duration of the tests?